scripts/vim-patch.sh

```diff
@@ -196,36 +213,49 @@ submit_pr() {
   check_executable hub
 
   cd "${NEOVIM_SOURCE_DIR}"
-  local checked_out_branch="$(git rev-parse --abbrev-ref HEAD)"
+  local checked_out_branch
+  checked_out_branch="$(git rev-parse --abbrev-ref HEAD)"
   if [[ "${checked_out_branch}" != ${BRANCH_PREFIX}* ]]; then
     echo "✘ Current branch '${checked_out_branch}' doesn't seem to be a vim-patch branch."
     exit 1
   fi
 
-  local git_remote=$(find_git_remote)
-  local pr_body="$(git log --reverse --format='#### %s%n%n%b%n' ${git_remote}/master..HEAD)"
-  local patches=("$(git log --reverse --format='%s' ${git_remote}/master..HEAD)")
+  local git_remote
+  git_remote=$(find_git_remote)
+  local pr_body
+  pr_body="$(git log --reverse --format='#### %s%n%n%b%n' "${git_remote}"/master..HEAD)"
+  local patches
+  patches=("$(git log --reverse --format='%s' "${git_remote}"/master..HEAD)")
   patches=(${patches[@]//vim-patch:}) # Remove 'vim-patch:' prefix for each item in array.
-  local pr_title="${patches[@]}" # Create space-separated string from array.
+  local pr_title=( "${patches[@]}" ) # Create space-separated string from array.
```

Comment:
This variable did not meant to be an array. I guess this should be

    local pr_title="${patches[*]}"